modules/assim.batch/tests/Rcheck_reference.log

```diff
@@ -140,9 +71,32 @@ File ‘PEcAn.assim.batch/R/pda.get.model.output.R’:
  OK
 * checking for unstated dependencies in vignettes ... OK
 * checking package vignettes in ‘inst/doc’ ... OK
-* checking re-building of vignette outputs ... OK
+* checking re-building of vignette outputs ... WARNING
+Error(s) in re-building vignettes:
+  ...
+--- re-building ‘AssimBatchVignette.Rmd’ using rmarkdown
+Error: processing vignette 'AssimBatchVignette.Rmd' failed with diagnostics:
+The function xfun::isFALSE() will be deprecated in the future. Please consider using base::isFALSE(x) or identical(x, FALSE) instead.
+--- failed re-building ‘AssimBatchVignette.Rmd’
+
+--- re-building ‘MultiSitePDAVignette.Rmd’ using rmarkdown
+Error: processing vignette 'MultiSitePDAVignette.Rmd' failed with diagnostics:
+The function xfun::isFALSE() will be deprecated in the future. Please consider using base::isFALSE(x) or identical(x, FALSE) instead.
+--- failed re-building ‘MultiSitePDAVignette.Rmd’
+
+SUMMARY: processing the following files failed:
+  ‘AssimBatchVignette.Rmd’ ‘MultiSitePDAVignette.Rmd’
+
+Error: Vignette re-building failed.
+Execution halted
+
```

Comment:
@moki1202 I think we can manually re-delete this warning -- I'm still not fully sure why xfun::isFALSE thing keeps coming back, but we can deal with it separately without having it in Rcheck_reference.log.
```suggestion
* checking re-building of vignette outputs ... OK

```